docs/example.html

```diff
@@ -8,6 +8,10 @@
     .exercise {
       margin: 50px;
     }
+
+    [data-datacamp-exercise] {
+        visibility: hidden;
```

Comment:
Visibility hidden is not necessary the best in your case. The widget will still take some the place of the text until it's loaded

Reply:
I used `display: none` at first, but i didn't like it when exercises loaded they moved everything on the webpage.

Reply:
Sorry, your comment is not really clear for me. I guess with this solution, there is an empty space with the size of the text to define the exercise. When the JS file is loaded, it's replaced by the widget. The only thing I don't like is the fact another ticket could be created after your fix saying to remove this empty space... IMHO, acceptance criteria are not well defined...

Reply:
I guess this is personal preference. Who should i ping for this?